src/FINDOLOGIC/Api/Responses/Xml21/Properties/Product.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Product
+{
+    /** @var string $id */
+    private $id;
+
+    /** @var float $relevance */
+    private $relevance;
+
+    /** @var array $properties */
+    private $properties = [];
+
+    /**
+     * Product constructor.
+     * @param SimpleXMLElement $result
+     */
+    public function __construct($result)
```

Comment:
Please typehint this with `SimpleXMLElement` and drop the PHPDoc as it is not really needed anyway.